src/pyaro/timeseries/Filter.py

```diff
@@ -1208,21 +1208,48 @@ def filter_stations(self, stations: dict[str, Station]) -> dict[str, Station]:
 @registered_filter
 class ValleyFloorRelativeAltitudeFilter(StationFilter):
     """
-    :param topo_file: Topography file path. Must be a dataset openable by xarray, with latitude
-        and longitude stored as "lat" and "lon" respectively. The variable that contains elevation
-        data is assumed to be in meters.
+    :param topo: Topography file path (either a file or a directory). Must be a dataset openable by
```

Comment:
Maybe you can add a short general explanation of the filter, e.g. derived from David description:

Several papers (eg Fowler et al, Lloibl et al.) on this have defined relative altitude as the height above the lowest topography within 5km. This ensures that sites on plateaus are properly treated as “surface” sites, but sites in hilly or mountainous areas (e.g. Schauinsland) are mountain sites.
The algorithm is simple: hRel = difference between site altitude and lowest point within 5km of site coordinates. This is essentially comparing the site with the valley-floor altitude.